src/components/button/Button.js

```diff
@@ -1,5 +1,3 @@
-import "./Button.css";
-
-export default function Button({ name }) {
-  return <button className="button">{name}</button>;
+export default function Button({ name, className }) {
+  return <button className={className}>{name}</button>;
```

Comment:
```suggestion
  return <button>{name}</button>;
```
How about you solve the problem, that (i guess) you want to tackle with the classname a little bit differently. You can use the the props inside of a styled component. Refer to the styled components Docs for help :) : https://styled-components.com/docs/basics#passed-props

```
${props => props.primary && css`
    background: white;
    color: black;
  `}
```
The classname solution is probably faster, and not wrong, but i challenge you just for the purpose of practicing styled components :)

Reply:
Challenge accepted :D